### PR TITLE
My applications page

### PR DESCRIPTION
**Pull request by AymanAlqoqa**
### Related:
related to #28

### Summary:
My Applications page will show my applications cards, It uses offers card and pass my application data to it.

hint: new styling class is added to main app style 'page__container', just add it to any main page container.

### Notes:
Optional notes

### Checklist Criteria
- [x] **Branch is up to date with master**
- [x] **The project runs**
- - [x] You can download and install locally, and run the project locally
- - [x] All existing and new features run without bugs
- - [x] The build script runs without error
- [x] **Code is consistent**
- - [x] Linter is used, no error messages
- - [x] Code is consistent with agreed team style
- - [x] Naming of files is consistent with project (e.g. all files named with camelCase)
- [x] **No console logs**
- - [x] Console.log is only used for crucial system messages (e.g. which port server is on).
- - [x] Any console.logs used for debugging have been removed.

### CODE REVIEW

File: client/src/Components/MyApplications/style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.my-applications__title {
```
Comment (Angham116) [inline client/src/Components/MyApplications/style.css]:
```suggestion
.my-applications__title {
    color: rgba(51, 51, 51, 0.8);
    font-size: 32px;
    padding-bottom: 15px;
    margin-bottom: 1.3rem;
    border-bottom: 2px solid rgba(51, 51, 51, 0.8);
    width: fit-content; 
    margin-left: 7px;
}
```
 to be same as me and fatma done.

File: client/src/Components/MyApplications/index.jsx [hunk 2]
```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import { Container } from 'react-bootstrap';
+import ApplicationCard from '../CommonComponents/OfferCard';
+import applications from '../utils/myApplications';
```
Comment (ashatat) [inline client/src/Components/MyApplications/index.jsx]: you didn't push this file? so how can we test your PR?

File: client/src/Components/MyApplications/index.jsx [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+import './style.css';
 
 export default function MyApplications() {
```
Comment (ashatat) [inline client/src/Components/MyApplications/index.jsx]: I think you need to send a request to get my-application list!!? #41 
so you need `componentDidMounth` :thinking:

File: client/src/Components/MyApplications/index.jsx [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
     <>
-      <h1>my applications</h1>
-      <UnderConstruction />
+      <Container className="page__container">
+        <div className="my-applications__title">
+          <span>My Applications</span>
+        </div>
+        {applications
+          ? applications.map(item => {
+              if (item.status !== 'active') {
```
Comment (ashatat) [inline client/src/Components/MyApplications/index.jsx]: why did you filter out the `active` status? :thinking:

File: client/src/Components/MyApplications/style.css (hunk 1 above)
Comment (AymanAlqoqa) [reply client/src/Components/MyApplications/style.css]: ok, I'll do it, but please rely on figma prototype.

File: client/src/Components/MyApplications/index.jsx [hunk 5]
```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import { Container } from 'react-bootstrap';
+import ApplicationCard from '../CommonComponents/OfferCard';
+import applications from '../utils/myApplications';
+import './style.css';
```
[the remark this answers is not in the document]
Comment (AymanAlqoqa) [reply client/src/Components/MyApplications/index.jsx]: ok, thanks

File: client/src/Components/MyApplications/index.jsx (hunk 2 above)
Comment (AymanAlqoqa) [reply client/src/Components/MyApplications/index.jsx]: it is pushed in offer card component PR #61.
I'll push it again in this PR for review simplicity
thanks

File: client/src/Components/MyApplications/index.jsx (hunk 3 above)
Comment (AymanAlqoqa) [reply client/src/Components/MyApplications/index.jsx]: that's right, I proposed to make it when I joined front-end with back-end.
I'll change it now 
thanks

File: client/src/Components/MyApplications/index.jsx (hunk 4 above)
Comment (AymanAlqoqa) [reply client/src/Components/MyApplications/index.jsx]: ohh, I'll remove this condition
thanks

**Pull request merged**